test/p2p/connection_reuse.ts

```diff
@@ -0,0 +1 @@
+console.log('Test');
```

Comment:
Is this test going to be fleshed out in this PR? If not I figure it can be removed.

Reply:
removed